spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe 'Foods index page', :vcr do
+
+    it 'displays the total number of items returned by the search' do
```

Comment:
I would also add to these tests that once you click `search` you would `expect(current_path).to eq("/foods")`